elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -433,7 +433,12 @@ private Object deleteObjects(Environment context) {
         ConnectionContainer connection = (ConnectionContainer) fetchObjects(context);
         Set<PersistentResource> toDelete = connection.getPersistentResources();
         toDelete.forEach(PersistentResource::deleteResource);
-        return new ConnectionContainer(toDelete, Optional.empty(), connection.getTypeName());
+
+        return new ConnectionContainer(
+                Sets.difference(connection.getPersistentResources(), toDelete),
```

Comment:
This doesn't look correct. I think your tests work because you've deleted everything. However,

```
toDelete = connection.getPersistentResources()
```

`Sets.difference()` will always be empty. If we want to return the _current_ state, we'll need to go fetch all of the objects without the id fields first.